src/cmds/backup.ts

```diff
@@ -1,9 +1,10 @@
 import yargs from 'yargs';
 import chalk from 'chalk';
 import { environmentConfigExists, getEnvironmentsConfig } from '../utils/environmentUtils';
-import { CleanService, ExportService, ImportService, ZipService } from '@kentico/kontent-backup-manager';
+import { CleanService, ExportService, ImportService, ZipService, IProcessedItem } from '@kentico/kontent-backup-manager';
 import { getFileBackupName } from '../utils/fileUtils';
-import { IProcessedItem } from '@kentico/kontent-backup-manager/_commonjs/src';
+// TODO - decide whether to use thi implementation or i.e. fs module or any other standard
```

Comment:
It could be omitted because MAPI JS SDK is a login error by default.

Decide where we need additional logging under debug flag.